src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -143,6 +145,13 @@ def get_args():
             "flags": ["--debug"],
             "action": "store_true",
             "help": "debug output",
+    }, {
+            "flags": ["--tabular-summary-print-fmt"],
```

Comment:
Can I suggest a shorter name? `--tabular-format` or `--summary-format`?

Reply:
In fact, this seems like a flag that users might actually want to tweak. Suggest having a short option too.